src/http_miou_client.ml

```diff
@@ -98,9 +99,12 @@ let http_1_1_response_handler ~f acc =
   (response_handler, response, acc)
 
 let http_1_1_error_handler response err =
-  let err =
-    match err with `Exn (Runtime.Flow msg) -> `Protocol msg | err -> `V1 err
+  let fn = function
+    | `Exn (Runtime.Flow msg) -> `Protocol msg
+    | err -> `V1 err
   in
+  let err = fn err in
+  Log.err (fun m -> m "%a" pp_error err);
```

Comment:
Here I would consider using warning level or use a different log source. In some cases it is not unexpected a remote is broken from time to time, and in that case it wouldn't be a real error. Then it would be nice to be able to mute such warnings/errors (but keep other errors)

Reply:
Do you have a precise idea of what we should do?

Reply:
Sorry I actually don't understand what sort of errors can happen here. I guess it's fine then.

Reply:
So it can be a protocol error (like, `h2` can say that the request is actually bad). User's exception are also retrieved and appear here (for instance, if the user raise an exception into the body consumer). The last kind is a TCP/IP and/or a TLS error.